models/api.model.js

```diff
@@ -0,0 +1,17 @@
+const db = require("../db/connection");
+
+exports.selectAllTopics = () => {
+  return db.query("SELECT * FROM topics").then((res) => {
+    return res.rows;
+  });
+};
+
+exports.selectArticleById = async (article_id) => {
```

Comment:
Good practice of async/await, however I would suggest that you stay consistent; if you're using async/await in your models, it's best practice to use it with all of it!